mods/ra/maps/allies-03b/allies03b.lua

```diff
@@ -366,4 +366,59 @@ WorldLoaded = function()
 	InitObjectives()
 	InitTriggers()
 	SetupAlliedUnits()
+
+	AmmoPoolTest()
+end
+
+AmmoPoolTest = function()
+	local yak = Actor.Create("yak", true, { Owner = player, Location = Tanya.Location + CVec.New(1, 1) })
+
+	Media.DisplayMessage("Ammo count: " .. yak.AmmoCount())
```

Comment:
It would be good if you could also display the max value so the full ammo checks are easier to make sense of.